r/DESCRIPTION

```diff
@@ -60,6 +60,7 @@ Collate:
     'chunked-array.R'
     'io.R'
     'compression.R'
+    'scalar.R'
```

Comment:
Is there a reason for the sorting here?

Reply:
Yeah, the `@include`s in compute.R. Scalar class should be defined before defining methods for it, so this Collate field controls the order in which the .R files are loaded in the package.